sim_instr/src/sim_instr_node.cpp

```diff
@@ -4,13 +4,16 @@
 #include <string>
 #include <fstream>
 #include <sstream>
-
+#include <iostream>
 
 struct TimeAndCmdVel{
     ros::Duration time;
     geometry_msgs::Twist cmd_vel;
 };
 
+/*bool compareByLength(const  TimeAndCmdVel &t1, const TimeAndCmdVel &t2) {
```

Comment:
don't commit commented out code - just delete it. that's why we have source control